Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Gallatross.xml

```diff
@@ -29,12 +29,12 @@
 			</li>
 
 			<li Class="PatchOperationReplace">
-		                 <xpath>/Defs/ThingDef[defName="AA_Gallatross"]/race/baseHealthScale</xpath>
-		                 <value>
-			               <baseHealthScale>15</baseHealthScale>
-		                 </value>
+				<xpath>/Defs/ThingDef[defName="AA_Gallatross"]/race/baseHealthScale</xpath>
+				<value>
+					<baseHealthScale>30</baseHealthScale>

```

Comment:
So, what exactly is the "recommended" approach to killing one of these things supposed to be if we're making it this tough? This this is already proof against most rifle rounds.